pending_changelog/monitorupdatingpersister.txt

```diff
@@ -0,0 +1,6 @@
+## Backwards Compatibility
+
+* Monitors stored with [`MonitorUpdatingPersister`] have a prepended sentinel value that prevents them from being deserialized by previous `Persist` implemtentions. This is to ensure that they are not accidentally read and used while pending updates are still stored separately, as this could result in penalty transactions. Users who wish to downgrade should perform the following steps:
+  * Make a backup copy of all channel state.
+  * Ensure all updates are applied to the monitors. This may require starting the node with 0.0.117 and immediately performing an orderly shutdown, or running a small crafted utility program. Restricting your node's network access during this process is a good precaution.
+  * Once all monitors have all updates applied, remove the sentinel byte sequence from the front of the monitor. For example, on a filesystem, this can be done with a small crafted utility program, or from Linux with the `dd` utility, see: https://unix.stackexchange.com/questions/6852/best-way-to-remove-bytes-from-the-start-of-a-file
```

Comment:
Yes, some other place I had this realization with @tnull. Much easier than removing the bytes offline. I offered to write this as a utility function, and maybe that's not necessary, but it could be useful for testing to do that.

Reply:
revised to suggest this, still an exercise for the reader at this point